aspnetcore/includes/not-latest-versionNM.md

```diff
@@ -2,6 +2,7 @@
 
 > [!NOTE]
 > This isn't the latest version of this article. To switch to the latest, use the ASP.NET Core version selector at the top of the table of contents.
+> Not all internal links work in older versions. If a link doesn't work, try the latest version.
```

Comment:
Just a suggestion, feel free to take or leave.

```suggestion
> Links to internal anchors might not work in older versions. If a link doesn't go to the intended section of the page, try the latest version.
```

Reply:
I suppose these lines will render as paragraphs without an extra blank line (`>`) between lines 4 and 5, but I usually drop them in anyway for consistency and clarity in markdown.